server/ansible/pbench-server-install.yml

```diff
@@ -1,7 +1,9 @@
 ---
+- name: install pbench RPM repo
+  import_playbook: pbench-repo-install.yml
```

Comment:
That's interesting; I'd love to see some history on this. Much of Ansible still baffles me; and in this case, the use of `pbench-repo-install` as both playbook and role probably doesn't help. But, essentially, you're changing the server install playbook from using just the `pbench-repo-install` __role__ to using the entire playbook, presumably to get the "become" behavior? Huh.

Reply:
No, to avoid have the `pbench-repo-install` role used in two places.

Another option is to just drop that mechanism entirely from the `pbench-server-install` playbook.

I don't believe it is helpful to have it by default because it presumes everybody wants to use the same COPR repo.